### DISCUSSION TITLE
What code should I adjust to remove AutoFocusPlugin?

### DISCUSSION THREAD

**Discussion by Michael-indochat** [sodenn/lexical-beautiful-mentions#156] [Q&A]
My page is about a form page, and the fields need to be filled in order from top to bottom, and my editor is at the bottom column, if AutoFocus is used, it will automatically slide to the bottom, but I don't want this , so I need to remove the AutoFocusPlugin inside the editor.

But at this time, after I removed AutoFocusPlugin, this error appeared when i first time to click insertMention, how can I fix it?

![image](https://github.com/sodenn/lexical-beautiful-mentions/assets/114893149/51885136-1770-4e96-9c47-c70cc311fb5c)

**Comment by Michael-indochat** [comment]
I find the error comes from here:
- path: https://github.com/sodenn/lexical-beautiful-mentions/blob/main/plugin/src/mention-commands.ts#L171
- line: 171 => `selection.insertNodes(nodes);`

**Reply by Michael-indochat** [reply]
@sodenn I hope you can help me to see how to solve this problem, thanks a lot !

**Reply by sodenn** [reply]
Hi @Michael-indochat, I think this is a bug in the mention plugin. I'll fix it in the next release. Thanks for reporting it!
